packages/components/tabs/src/tabs.tsx

```diff
@@ -25,7 +25,7 @@ import type { TabsPaneContext } from '@element-plus/tokens'
 import type { ExtractPropTypes } from 'vue'
 import type { Awaitable } from '@element-plus/utils'
 
-export type TabPanelName = string | number
+export type TabPanelName = string | number | undefined
```

Comment:
This type should not include undefined.

Reply:
Can I use `as TabPanelName`?

Reply:
I do not recommend you do this

When running the function, these function parameters should all have values, the TS does not know this.

```ts
    const setCurrentName = async (value?: TabPanelName) => {
      if (currentName.value === value || isUndefined(value)) return
```

Reply:
just setCurrentName and handleTabRemove function